### PR TITLE
Update docker-compose.prod.yaml

### PR DESCRIPTION
**Pull request by Tshibek**
Update to deployapp with caddy

### CODE REVIEW

**Comment by sobolevn** [comment]
Looks like this check fails: https://github.com/wemake-services/wemake-django-template/blob/master/tests/test_github.sh#L36-L38

Can you please see why?

**Comment by Tshibek** [comment]
well if i will know, i will tell you hah,

`Creating wemake-django-template_web_1 ... 
Creating wemake-django-template_web_1 ... done
Creating wemake-django-template_caddy_run ... 
Creating wemake-django-template_caddy_run ... done
Error: decoding config: unexpected end of JSON input
1
Error: Process completed with exit code 1.`

**Comment by Tshibek** [comment]
the problem is in ci.sh with path, i changed it localy to /etc/caddy/Caddyfile and still is the same problem, but when back to /srv/CaddyFile back normal idk why

**Comment by sobolevn** [comment]
It is hardcoded here https://github.com/wemake-services/wemake-django-template/blob/master/%7B%7Bcookiecutter.project_name%7D%7D/docker/caddy/ci.sh#L13

**Comment by sobolevn** [comment]
Sorry, why did you close this PR? :)

**Comment by Tshibek** [comment]
Miss click, i changed again ci.sh, now it work locally, but i dont know is acceptable

File: {{cookiecutter.project_name}}/docker/caddy/ci.sh [hunk 1]
```diff
@@ -6,15 +6,15 @@ set -o nounset
 
 run_ci () {
   # Validating:
-  caddy validate
+  caddy validate --config /etc/caddy/Caddyfile
```
Comment (sobolevn) [inline {{cookiecutter.project_name}}/docker/caddy/ci.sh:9]: Let's define `CADDYFILE='/etc/caddy/Caddyfile'` at the top of the file and then use it: 

```suggestion
  caddy validate --config "$CADDYFILE"
```

**Review by sobolevn — COMMENTED** [review]
Awesome, only one minor nit left :)

File: {{cookiecutter.project_name}}/docker/caddy/ci.sh (hunk 1 above)
Comment (sobolevn) [reply {{cookiecutter.project_name}}/docker/caddy/ci.sh:9]: And in other places

Comment (Tshibek) [reply {{cookiecutter.project_name}}/docker/caddy/ci.sh]: should be good now, commited

**Review by sobolevn — APPROVED** [review]
Thank you!

**Pull request merged**